packages/integrations/image/README.md

```diff
@@ -22,8 +22,22 @@ This integration provides `<Image />` and `<Picture>` components as well as a ba
 
 ## Installation
 
+Along with our integration, we recommend installing [sharp](https://sharp.pixelplumbing.com/) when appropriate. 
 
- ### Quick Install
+The `@astrojs/image` default image transformer is based on [Squoosh](https://github.com/GoogleChromeLabs/squoosh) and uses WebAssembly libraries to support most deployment environments, including those that do not support sharp, such as StackBlitz and Netlify Edge.
+
+For faster builds and more fine-grained control of image transformations, install sharp in addition to `@astrojs/image` if
+- You are building a static site with Astro.
+- You are using an SSR deployment host that supports NodeJS using `@astrojs/netlify/functions`, `@astrojs/vercel/serverless` or `@astrojs/node`.
+
+
+Note that `@astrojs/image` is not currently supported on
+- Cloudflare SSR
+- @astrojs/deno
+- @astrojs/vercel/edge
+
```

Comment:
`@astrojs/netlify/edge-functions` probably belongs here as well since it runs on deno